floris/tools/floris_interface.py

```diff
@@ -801,6 +801,35 @@ def get_farm_power(self,
             ]
             return np.sum(turb_powers)
 
+    def get_turbine_layout(self, z=False):
```

Comment:
Lines 1200 and 1214 define properties `layout_x` and `layout_y`, respectively. Perhaps adding another property, `layout_z`, and then allowing the user to grab what they want (e.g. `layout_x, layout_y = fi.layout_x, fi.layout_y`)? Or is it worth it to have a helper method that returns the properties?

Reply:
Ah. I was using tab completion on the `FlorisInterface` object for functions starting with "get", which returned several functions involving `get_turbine_*`. I didn't see anything pertaining to the layout, which is why I added this.  To me, having this as a property is less intuitive--I don't expect an "interface" to have a "layout".

Reply:
@bayc Up to you how to proceed, both code solutions work for me.

Reply:
@ewquon Gotcha, yea, I can see how that makes sense. It would be nice to a standard `get_turbine_` prefix for these methods. @paulf81 and @rafmudaf , what do you think?

Reply:
this looks good to me, we might want to make a check through the code for places where we currently dig for the layout and instead use this function for consistency?